Core/HLE/sceFont.cpp

```diff
@@ -1100,6 +1272,24 @@ static int sceFontGetCharInfo(u32 fontHandle, u32 charCode, u32 charInfoPtr) {
 	auto charInfo = PSPPointer<PGFCharInfo>::Create(charInfoPtr);
 	font->GetCharInfo(charCode, charInfo);
 
+	u32 allocSize = charInfo->bitmapWidth * charInfo->bitmapHeight;
+	if (charInfo->sfp26AdvanceH != 0 || charInfo->sfp26AdvanceV != 0) {
+		if (font->GetFontLib()->GetCharInfoBitmapAddress() != 0) {
```

Comment:
Hm, so in theory this alternates between allocating and freeing each call?  That seems like odd behavior.

-[Unknown]

Reply:
The alloc size will always be different if the char is different, so this behavior seems reasonable?

Reply:
Well, that part makes sense, but I'd expect it to likely allocate AND deallocate for each call?  I guess would have to test to see what it does.  But this:

sceFontGetCharInfo('A'); // calls allocate, keeps A allocated
sceFontGetCharInfo('B'); // calls free on A, nothing allocated now
sceFontGetCharInfo('C'); // calls allocate, keeps C allocated

Seems like strange behavior - you'd think it'd be more like:

sceFontGetCharInfo('A'); // calls allocate, keeps A allocated
sceFontGetCharInfo('B'); // calls free on A, calls alloc on B
sceFontGetCharInfo('C'); // calls free on B, calls alloc on C

Or:

sceFontGetCharInfo('A'); // calls alloc on A, then free on A
sceFontGetCharInfo('B'); // calls alloc on B, then free on B
sceFontGetCharInfo('C'); // calls alloc on C, then free on C

-[Unknown]